src/um/pdh.rs

```diff
@@ -54,3 +222,338 @@ STRUCT!{struct PDH_COUNTER_PATH_ELEMENTS_W {
+    phDataSource: *mut PDH_HLOG, szLogFileNameList: LPCSTR
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhOpenQueryH(
+    hDataSource: PDH_HLOG, dwUserData: DWORD_PTR, phQuery: *mut PDH_HQUERY
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumMachinesHW(
+    hDataSource: PDH_HLOG, mszMachineNameList: LPWSTR, pcchBufferLength: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumMachinesHA(
+    hDataSource: PDH_HLOG, mszMachineNameList: LPSTR, pcchBufferLength: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectsHW(
+    hDataSource: PDH_HLOG, szMachineName: LPCWSTR, mszObjectList: LPWSTR,
+    pcchBufferLength: LPDWORD, dwDetailLevel: DWORD, bRefresh: BOOL
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectsHA(
+    hDataSource: PDH_HLOG, szMachineName: LPCSTR, mszObjectList: LPSTR,
+    pcchBufferLength: LPDWORD, dwDetailLevel: DWORD, bRefresh: BOOL
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectItemsHW(
+    hDataSource: PDH_HLOG, szMachineName: LPCWSTR, szObjectName: LPCWSTR,
+    mszCounterList: LPWSTR, pcchCounterListLength: LPDWORD, mszInstanceList: LPWSTR,
+    pcchInstanceListLength: LPDWORD, dwDetailLevel: DWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhEnumObjectItemsHA(
+    hDataSource: PDH_HLOG, szMachineName: LPCSTR, szObjectName: LPCSTR,
+    mszCounterList: LPSTR, pcchCounterListLength: LPDWORD, mszInstanceList: LPSTR,
+    pcchInstanceListLength: LPDWORD, dwDetailLevel: DWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhExpandWildCardPathHW(
+    hDataSource: PDH_HLOG, szWildCardPath: LPCWSTR, mszExpandedPathList: LPWSTR,
+    pcchPathListLength: LPDWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhExpandWildCardPathHA(
+    hDataSource: PDH_HLOG, szWildCardPath: LPCSTR, mszExpandedPathList: LPSTR,
+    pcchPathListLength: LPDWORD, dwFlags: DWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetDataSourceTimeRangeH(
+    hDataSource: PDH_HLOG, pdwNumEntries: LPDWORD, pInfo: PPDH_TIME_INFO,
+    pdwBufferSize: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetDefaultPerfObjectHW(
+    hDataSource: PDH_HLOG, szMachineName: LPCWSTR, szDefaultObjectName: LPWSTR,
+    pcchBufferSize: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetDefaultPerfObjectHA(
+    hDataSource: PDH_HLOG, szMachineName: LPCSTR, szDefaultObjectName: LPSTR,
+    pcchBufferSize: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetDefaultPerfCounterHW(
+    hDataSource: PDH_HLOG, szMachineName: LPCWSTR, szObjectName: LPCWSTR,
+    szDefaultCounterName: LPWSTR, pcchBufferSize: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhGetDefaultPerfCounterHA(
+    hDataSource: PDH_HLOG, szMachineName: LPCSTR, szObjectName: LPCSTR,
+    szDefaultCounterName: LPSTR, pcchBufferSize: LPDWORD
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhBrowseCountersHW(
+    pBrowseDlgData: PPDH_BROWSE_DLG_CONFIG_W
+) -> PDH_STATUS}
+EXTERN!{stdcall fn PdhBrowseCountersHA(
+    pBrowseDlgData: PPDH_BROWSE_DLG_CONFIG_A
```

Comment:
`PdhBrowseCountersHA` and `PdhBrowseCountersHW` seem to take [PPDH_BROWSE_DLG_CONFIG_H*](https://msdn.microsoft.com/en-us/library/windows/desktop/aa373034(v=vs.85).aspx) not PPDH_BROWSE_DLG_CONFIG_*